extension/src/Runner/testRunnerWrapper.ts

```diff
@@ -121,4 +124,31 @@ export class TestRunnerWrapper {
             duration: notRun ? undefined : duration.toString(),
         };
     }
+
+    private static execPreLaunchTask(cwd: string, task: string): Promise<any> {
+        return new Promise((resolve, reject) => {
+            const process = cp.exec(task, { maxBuffer: Configs.CHILD_PROCESS_MAX_BUFFER_SIZE, cwd });
+            process.on('error', (err) => {
+                Logger.error(
+                    `Error occurred while executing prelaunch task. Name: ${err.name}. Message: ${err.message}. Stack: ${err.stack}.`,
+                    {
+                        stack: err.stack,
+                    });
+                reject(err);
+            });
+            process.stderr.on('data', (data) => {
+                Logger.error(`Error occurred: ${data.toString()}`);

```

Comment:
Should output to let user know the process information of process?

Reply:
it would be output to output channel